core/trino-main/src/test/java/io/trino/security/TestAccessControlManager.java

```diff
@@ -209,6 +209,7 @@ public void testDenyTableFunctionCatalogAccessControl()
         }
     }
 
+    // TODO: need to properly handle the ordering of multiple masks as they are not allowed currently
```

Comment:
I think the TODO and test name is incorrect. I would simply remove TODO and name test to `testMultipleColumnMasks`